client/lib/post-types-list/list.js

```diff
@@ -2,6 +2,8 @@
  * External dependencies
  */
 var debug = require( 'debug' )( 'calypso:post-types-list' );
+var omit = require( 'lodash/object/omit' );
```

Comment:
lodash 4 compat warning. you'll have to drop the middle value on the path and rebase.